src/main/java/com/microsoft/graph/serializer/DerivedClassIdentifier.java

```diff
@@ -39,11 +46,7 @@ public DerivedClassIdentifier(@Nonnull ILogger logger) {
             /** #microsoft.graph.user or #microsoft.graph.callrecords.callrecord */
             final String odataType = jsonObject.get(ODATA_TYPE_KEY).getAsString();
             final int lastDotIndex = odataType.lastIndexOf(".");
```

Comment:
```suggestion
```

Reply:
This line should be removed since lastDotIndex is no longer being used here. This will resolve spotbugs warning.

Reply:
comment addressed in #310